### ISSUE TITLE
During PIN/Fingerprint Unlock -- Do not show Account Details/Balance

### ISSUE THREAD

**Issue by dmo580** [bitpay/copay#5958]
When first opening Copay from a cold launch, it first loads a splash screen where you're asked for your PIN/Fingerprint. This is good.

However if the app is still in memory and you are returning to it, you're shown the previous screen which could include balances and transaction history. While app functionality is not allowed until you unlock it via fingerprint/PIN, for privacy purposes to force the user back into a splash screen showing that the app is locked before prompting for security details.

Otherwise users would have to remember to explicitly swipe the app away from memory.

**Comment by dabura667** [comment]
This can be done by placing a splash screen in the foreground before the user blurs the app. I would assume most frameworks support an onBlur event of some kind.

+1

**Comment by gabrielbazan7** [comment]
Yes, we have plans to improve the pin feature in the future.

**Comment by dmo580** [comment]
On a similar note can you explain what the "Use Backup" button does on Android? It's very unclear and it doesn't seem to work
![screenshot_20170504-014327](https://cloud.githubusercontent.com/assets/1091670/25716527/301be63c-30b4-11e7-9257-0c150c3fafae.png)

**Comment by gabrielbazan7** [comment]
use backup if I'm not wrong is to use the code/pattern from your phone.

**Comment by dmo580** [comment]
That's what I thought. What happens is the Android screen asking for my Password/PIN/Fingerprint comes up. I enter that, and it brings me back to Copay's screen shown above. It's an endless loop meaning the backup doesn't work.

Either way I think this screen needs additional clarification. There's many passwords/passphrases out there--one for our devices, one for spending password, BIP39 Passphrase. There needs to be a consistent way to tell a user which one we want.

**Comment by cmgustavo** [comment]
Closes #5986

**Comment by dmo580** [comment]
Hi @cmgustavo I noticed the recent update does address this however it's not 100% perfect. When I switch back into Copay from recent apps in Android, the screen still initially shows the previous state of the app before bringing me back into the splash screen.

I compared this with LastPass which I have set to always require password/fingerprint, and regardless of my previous screen in LastPass, it will relaunch in a splash screen/protected page to hide the previous contents. In fact LastPass appears as a blank screen even in the recents menu so it protects all content


![](https://i.imgur.com/LbVmRjQ.png)
